ArchiSteamFarm/IPC.cs

```diff
@@ -1226,6 +1323,19 @@ private sealed class GamesToRedeemInBackgroundRequest {
 			private GamesToRedeemInBackgroundRequest() { }
 		}
 
+		private sealed class GamesToRedeemInBackgroundResponse {
+			[JsonProperty(Required = Required.Always)]
+			internal readonly OrderedDictionary UsedKeys;
+
+			[JsonProperty(Required = Required.Always)]
+			internal readonly OrderedDictionary UnusedKeys;
+
+			internal GamesToRedeemInBackgroundResponse(OrderedDictionary used, OrderedDictionary unused) {
```

Comment:
If you name your main property `UsedKeys` then for consistency this one should be `usedKeys`. Also you shouldn't declare `Required` for objects we're going to serialize ourselves.

Reply:
sorry. forgot to rename that after i renamed the property